webapp/channels/src/packages/mattermost-redux/src/selectors/entities/posts.ts

```diff
@@ -61,6 +61,18 @@ export function getPostsInThread(state: GlobalState): RelationOneToMany<Post, Po
     return state.entities.posts.postsInThread;
 }
 
+export function getPostsInThreadOrdered(state: GlobalState, rootId: string): string[] {
```

Comment:
I'm really surprised that these posts aren't already ordered in `state.entities.postsInThread`. Posts in each chunk in `postsInChannel` are ordered, so I would've expected the ones in the thread to be too